### DISCUSSION TITLE
Lesson 6 Part 14: Problem: Source "lib/forge-std/src/Script.sol" not found: File not found. Searched the following locations: "".solidity(6275)

### DISCUSSION THREAD

**Discussion by wuuer** [Cyfrin/foundry-full-course-cu#2095] [Q&A]
Hi, I have a problem with Deploy.sol
vscode shows problem:
Source "lib/forge-std/src/Script.sol" not found: File not found. Searched the following locations: "".
Source "src/SimpleStorage.sol" not found: File not found. Searched the following locations: "".
```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.19;

import {Script} from "forge-std/Script.sol";
import {SimpleStorage} from "../src/SimpleStorage.sol";

contract Deploy is Script {
    function run() external returns (SimpleStorage) {}
}

```
But, I can jump to the Script.sol file by pressing F12 and compile with no errors.

And the sol files in lib/forge-std/src also have this problem.

is it a bug of vscode extension NomicFoundation.hardhat-solidity ?

![error](https://github.com/user-attachments/assets/e5d7348f-79df-4c5d-be81-77cb7468e625)

**Comment by EngrPips** [comment]
Sometimes, `vscode` misbehaves and loads slowly; you should close your `vscode` and reopen it, and if the problem persists but doesn't affect you compiling your contracts, you should continue and ignore it.

**Comment by hgky95** [comment]
It looks like you have an issue with remapping. Please check this document [Foundry remapping](https://book.getfoundry.sh/projects/dependencies#remapping-dependencies)
Correct the remapping then close/re-open the VSCode.

**Comment by EngrPips** [comment]
Hey @wuuer, have you been able to resolve this already?

**Reply by wuuer** [reply]
No. Just ignore it

**Reply by EngrPips** [reply]
I see, That means it does not block your development process.

**Reply by s3bc40** [reply]
@EngrPips the solution of @hgky95 is working if we apply what they recommend in the doc : 

`forge remappings` 

https://book.getfoundry.sh/projects/dependencies#remapping-dependencies

Then we just need to restart vscode or vscodium, I verified afterward by `ctrl+click` on `Script.sol`.

Thanks @hgky95

**Reply by EngrPips** [reply]
Awesome. This is interesting.

**Comment by s3bc40** [comment]
Hey :wave:  

When I arrived at this stage of the course, I checked my lib and I got nothing inside. It might have been a problem on my side while installing (sorry).

But if you have the same issue, and you have nothing in your `/lib` dir, you can use this solution : https://ethereum.stackexchange.com/a/153108

And if you have the same error with the forge install, you can run this command in your workdir : 

```shell
forge install --no-commit foundry-rs/forge-std
``` 

The error : 
![image](https://github.com/user-attachments/assets/71346617-7e99-4428-a0bd-2187587ca4e5)

**Reply by EngrPips** [reply]
Thanks for the heads-up.
